src/examples/button/IconButtonDefault.tsx

```diff
@@ -9,13 +9,16 @@ import React from 'react';
 import { Row, Col } from '@zendeskgarden/react-grid';
 import { IconButton } from '@zendeskgarden/react-buttons';
 import { ReactComponent as LeafIcon } from '@zendeskgarden/svg-icons/src/16/leaf-stroke.svg';
+import { Tooltip } from '@zendeskgarden/react-tooltips';
 
 const Example = () => (
   <Row>
     <Col textAlign="center">
-      <IconButton aria-label="leaf">
-        <LeafIcon />
-      </IconButton>
+      <Tooltip content="Leaf">
```

Comment:
Updated examples to follow Tooltip pattern for icon-only buttons.

Reply:
Hey @smritimadan I'm sorry to say I think I got this one wrong. Because of the current `Tooltip` behavior which adds a `aria-describedby` attribute to its child, I'm recently convinced that this isn't an _either-or_ but rather a _both-and_ situation. These `IconButton` elements should have both a `Tooltip` parent _and_ a `aria-label` attribute added directly to the button. The voiceover experience is slightly enhanced, and we solve for a11y violations seen here, for example: https://zendeskgarden.github.io/react-components/?path=/story/components-colorpickers-colorswatch--uncontrolled

Reply:
Oop! I will update all of these. Thank you!